webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -110,7 +110,7 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
           handleError(e, {
             level: ErrorLevel.Error,
             publicMessage: 'Failed to fetch list of workspace namespace bindings.',
-            silent: false,
+            silent: true,
```

Comment:
**question:** this effectively silences this error for good from the user perspective but will still get logged, confirming that it is what we want (as opposed to conditionally showing it to the user)

Reply:
cc @salonig23